uwblueprint/community-fridge-kw
# feat: admin export food rescues and checkins

## Brief description. What is this change? 

### [Admin Export All Fridge Check-ins/Food Rescues](https://www.notion.so/uwblueprintexecs/Admin-Export-All-Fridge-Check-ins-Food-Rescues-262515bd7cbe43ceb5111eb9066c8567)



## Implementation description. How did you make this change?
* Added `generateCSV` utils file back



## Steps to test
1. Create some checkins and schedules
2. Go to http://localhost:3000/admin/check-ins and click on export. It should download a csv file. Verify that the data outputted in the csv file is accurate.
3. Go to http://localhost:3000/admin/view-donations and click on export. It should download a csv file. Verify that the data outputted in the csv file is accurate.




## Checklist
